src/OpenTelemetry.Api/Trace/ActivityExtensions.cs

```diff
@@ -39,6 +39,7 @@ public static class ActivityExtensions
     /// <param name="activity">Activity instance.</param>
     /// <param name="status">Activity execution status.</param>
     [MethodImpl(MethodImplOptions.AggressiveInlining)]
+    [Obsolete("This is deprecated after DiagnosticSource 6.0, see https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Api#setting-status")]
```

Comment:
A few suggestions...

* Tell people what to call instead in the message.
* Clean up the summary.
* Move the link to `remarks`.

Something like this...

```diff
    /// <summary>
    /// Sets the status of activity execution.
-   /// Activity class in .NET does not support 'Status'.
-   /// This extension provides a workaround to store Status as special tags with key name of otel.status_code and otel.status_description.
-   /// Read more about SetStatus here https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#set-status.
    /// </summary>
+   /// <remarks>
+   /// For more details see: <see href="https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Api#setting-status" />.
+   /// </remarks> 
    /// <param name="activity">Activity instance.</param>
    /// <param name="status">Activity execution status.</param>
    [MethodImpl(MethodImplOptions.AggressiveInlining)]
+   [Obsolete("Call Activity.SetStatus instead. SetStatus will be removed in a future version.")]
    public static void SetStatus(this Activity activity, Status status)
    {
    }
```

PS: If we are going to do this, we should probably also do it to `GetStatus`. Doesn't have to be on this PR.

Reply:
> * Tell people what to call instead in the message.

+100 👍

Reply:
Thanks @CodeBlanch !
This seems more clear!
I have corrected my PR.